app/src/main/java/net/bible/android/control/link/LinkControl.kt

```diff
@@ -255,6 +262,26 @@ class LinkControl @Inject constructor(
         }
     }
 
+    private fun showNote(osisRef: String?, note: String) {
+        val activity = CurrentActivityHolder.getInstance().currentActivity
+        val handlerIntent = Intent(activity, FootnoteAndRefActivity::class.java)
+        val intentHelper = IntentHelper()
+        val osisParser = OsisParser()
+        val verseRange = osisParser.parseOsisRef(this.currentPageManager.currentBible.versification, osisRef)
+        intentHelper.updateIntentWithVerseRange(handlerIntent, verseRange)
+        activity.startActivityForResult(handlerIntent, ActivityBase.STD_REQUEST_CODE)
+    }
+
+    private fun showMyNote(osisRef: String) {
+        val activity: MainBibleActivity = CurrentActivityHolder.getInstance().currentActivity as MainBibleActivity
+        val osisParser = OsisParser()
+        val verseRange = osisParser.parseOsisRef(this.currentPageManager.currentBible.versification, osisRef)
+        activity.fullScreen = false
+        currentPageManager.showMyNote(verseRange)
+        activity.invalidateOptionsMenu()
+        activity.documentViewManager.buildView()
+    }
+
```

Comment:
Currently there's a bit of code duplication between these functions and https://github.com/AndBible/and-bible/blob/13893812e8e7839de5b682f1370dddac8a13082b/app/src/main/java/net/bible/android/view/activity/page/actionmode/VerseMenuCommandHandler.java#L67, but I don't know what is a good place to put the common function.